packages/deposit_supabase/test/supabase_deposit_adapter_test.dart

```diff
@@ -51,6 +51,28 @@ void main() {
       });
     });
 
+    group('.addAll()', () {
+      test('add multiple items', () async {
+        await adapter.addAll(
+          'cars',
+          'id',
+          [
+            <String, dynamic>{'brand': 'VW', 'model': 'Nivus'},
+            <String, dynamic>{'brand': 'VW', 'model': 'Virtus'},
+          ],
+        );
+
+        final result = await client.from('cars').select().execute();
+        final data = (result.data as List).cast<Map>();
```

Comment:
This line looks weird, you first cast to `List` then to a `Map`?  Am I am missing something hahaha?

Reply:
It is a convoluted way to cast a `List<Map>`. If I did it as `result.data as List<Map>` it complains that a `List<dynamic>` cant be cast too `List<Map>`.